search-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import { RouteNames } from '@/enums'
+import { mount } from '@vue/test-utils'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { BcrsBreadcrumb } from '@/bcrs-common-components'
+import { SearchDashboardBreadcrumb, SearchSIGNINBreadcrumb } from '@/bcrs-common-components/resources'
+import store from '@/store'
+
+
+describe('Breadcrumb.vue', () => {
+
+  let wrapper;
+  const breadcrumbs : BreadcrumbIF[] = [
+    {
+      text: SearchSIGNINBreadcrumb.text,
+      to: { name: RouteNames.SIGN_IN }
+    },
+    {
+      text: SearchDashboardBreadcrumb.text,
+      to: { name: RouteNames.DASHBOARD }
+    }
+  ]
+
+  wrapper = mount(BcrsBreadcrumb, {
+    props: { breadcrumbs },
+    global: {
+      provide: {
+        store: store
+      },
+      mocks: {
```

Comment:
this isn't how you mock the router for the composition-api. See https://test-utils.vuejs.org/guide/advanced/vue-router.html#using-a-mocked-router-with-composition-api

Reply:
Updated. Thanks!